libvcx/src/utils/devsetup.rs

```diff
@@ -370,7 +370,11 @@ pub fn setup_indy_env(use_zero_fees: bool) {
 }
 
 pub fn cleanup_indy_env() {
-    delete_wallet(settings::DEFAULT_WALLET_NAME, None, None, None).unwrap();
+    let walletName = settings::DEFAULT_WALLET_NAME;
+    match delete_wallet(walletName, None, None, None) {
```

Comment:
This can be done on one line with something like
`delete_wallet(settings::DEFAULT_WALLET_NAME, None, None, None).unwrap_or_else(|_|  error!("Error deleting wallet {}", walletName));`, no?

Reply:
Good point Mr. Rustafarian it's about time to start writing proper idiomatic Rust hehe 👍 Fixed